conditions.go

```diff
@@ -166,6 +167,13 @@ func (a *Address) UnmarshalJSON(raw []byte) error {
 		}
 		*a = c.Address()
 		return nil
+	case "bech32":
+		_, addr, err := bech32.Decode(enc)
+		if err != nil {
+			return errors.Wrapf(err, "deserialize bech32: %s", err)
```

Comment:
q: why do you print the `err` with the message again?

Reply:
q: do we need to enforce a prefix check? `tiov` in this case?

Reply:
One reason we did not have bech32 in weave earlier is how we enforce prefix checks.

People may use weave with many possible prefixes. Let's not enforce any one of them.

Other solution would be to set a list of allowed bech32 prefixes somewhere in the app configuration (eg. `cmd/bnsd`) but since this is just for genesis file parsing, and we never export with this, I think we don't need that level (but feel free to convince me otherwise and add an issue to add enforcement)

Reply:
👍 we can expect the genesis to be correct, indeed.